### DISCUSSION TITLE
How to use github

### DISCUSSION THREAD

**Discussion by Mimi47** [community/community#21572] [New to GitHub]
<p>I don’t know how to use Github</p>

**Comment by Mimi47** [comment]
<p>:heart: I don’t know how to create I need help</p>

**Comment by airtower-luna** [comment]
<p>If you need help, please describe what you need help with: What is your goal, what have you tried already, what problems have you encountered? The more specific you are the more likely it is that someone will have an idea.</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hi there <a class="mention" href="/mimi47">@Mimi47</a> :wave:</p>
<p>Thanks so much for joining us here in the GitHub Support Community! A fantastic starting point would be our How to Use Git &amp; GitHub Guidepost, here:</p>
<aside class="quote quote-modified" data-post="1" data-topic="168441">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/the-how-to-use-git-github-guidepost/168441">round_pushpin The How to Use Git &amp; GitHub Guidepost</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    Just getting started with how to use Git &amp; GitHub? You’ve come to the right place. 
Our team curated a collection of topics, solutions and resources organized by intent to help you get started. world_map 
Getting started with <a href="http://GitHub.com">GitHub.com</a>


Resource: <a href="https://lab.github.com/">GitHub Learning Lab</a>

Topics

<a href="https://github.community/t/what-are-the-top-3-things-you-wish-you-knew-when-you-were-getting-started-with-git-and-github/160624">What are the top 3 things you wish you knew when you were getting started with Git and GitHub?</a>
<a href="https://github.community/t/startup-on-github/305/2">Startup on GitHub</a>

<a href="https://github.community/t/how-do-you-put-images-on-the-readme-md-file/576/2">How do you put images on the README.md file</a>?



File Management with Git &amp; GitHub

<a href="https://github.community/t/how-to-delete-multiples-files-in-github/702">How to delete multiple …</a>
  </blockquote>
</aside>

<p>Lots of helpful resources there, and if you have questions as you go through, you may want to search this forum to see if your question has been asked already. If not, pose your question in it’s appropriate category and typically, someone will be around to help out with your specific problem.</p>
<p>Cheers!</p>
